scripts/fuzz_shell.js

```diff
@@ -138,17 +138,10 @@ function logValue(x, y) {
   console.log('[LoggingExternalInterface logging ' + printed(x, y) + ']');
 }
 
-// Some imports need to access exports by index.
-var exportsList;
-function getExportByIndex(index) {
-  if (!exportsList) {
-    exportsList = [];
-    for (var e in exports) {
-      exportsList.push(e);
-    }
-  }
-  return exports[exportsList[index]];
-}
+// Track the exports in a map (similar to the Exports object from wasm) and also
+// in a list of names, as some imports need to access by index.
+var exports = {};
```

Comment:
It could be one I suppose, but we are only going to map strings here, which work either way. And a Map doesn't have operator overloading, i.e., we'd need to write `exports.get(x)` rather than `exports[x]`.